spec/lib/tasks/content_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe "content:export_results_to_csv" do
+  include_context "rake"
+
+  before do
+    allow(ContentExporter).to receive(:generate_results_link_csv).and_return("csv\n woop")
+  end
+
+  it "should include environment as a prerequsite" do
```

Comment:
Here `prerequisite` is misspelt as `prerequsite`.

Reply:
... :man_facepalming: thank you...